src/reducers/computeRenderedRows.js

```diff
@@ -207,35 +211,68 @@ function computeRenderedRowOffsets(state, rowRange) {
     return;
   }
 
-  const bufferMapping = []; // state.rows
+  // output for this function
+  const bufferMapping = state.rows;
```

Comment:
Why do we start from the old rows list here?  Is this just to ensure we stick to the same size / don't evict unless necessary?  

It seems a bit obscure of a way to accomplish that.  And I think it could end up w/ the same row rendered in 2 places (although that may not be problematic).

Reply:
Could you take a pass at cleaning it up to remove the obscurity for why it's here?  It's not clear why it's needed from the way the code is written, so I fear it will easily be broken.